frontend/src/streamlist/DeleteStream.jsx

```diff
@@ -24,7 +24,7 @@ export default function DeleteStream(props) {
     return setOpen(false);
   };
   const confirmDelete = () => {
-    StreamApi.deleteStream(deleteId, () => {
+    StreamApi.deleteStream(deleteId).then(() => {
       history.push("/Streaming");
       history.go(0);
```

Comment:
Delete stream file is deleted in #332 You can also delete it here if that helps with your test coverage numbers

Reply:
Rebased with master to get the changes from the refactor dialog PR, and the coverage has indeed increased 😄